src/lamplib/src/genny/tasks/dry_run.py

```diff
@@ -24,6 +24,7 @@ def dry_run_workload(
         "MixedWorkloadsGennyStress.yml",
         "ClusteredCollection.yml",
         "ClusteredCollectionLargeRecordIds.yml",
+        "ConnectionsBuildup.yml",
```

Comment:
Add ```#Todo: consider removing ConnectionsBuildup.yml once EVG-21054 is complete.```

Reply:
The SLOG.info right below this was updated in a previous PR to [reference EVG-21054](https://github.com/mongodb/genny/blob/master/src/lamplib/src/genny/tasks/dry_run.py#L29) rather than TIG-3290 (which has mysteriously disappeared/been deleted). When all workloads being merged as part of this ticket are finished there will be 10+ workloads added to this list so I think the reference to EVG-21054 is enough rather than a #TODO - I can also leave a comment on EVG-21054 to highlight the workloads that can be removed from here once it's complete.

Reply:
:+1: